### ISSUE TITLE
Run TOC Election for Jan 2019

### ISSUE THREAD

**Issue by caniszczyk** [cncf/toc#169]
TOC nominations are starting TODAY for the 6 TOC slots that open up in Jan 2019 that are selected by GB members (members can propose UP TO TWO NOMINEES, where you can only nominate up to one candidate from your organization, the other candidate has to be outside your organization): https://github.com/cncf/toc/blob/master/process/election-schedule.md

The seats that open up are:

Jonathan Boulle (term: 3 years - start date: 1/29/2016)
Bryan Cantrill (term: 3 years - start date: 1/29/2016)
Camille Fournier (term: 3 years - start date: 1/29/2016)
Benjamin Hindman (term: 3 years - start date: 1/29/2016)
Ken Owens (term: 3 years - start date: 1/29/2016)
Alexis Richardson (term: 3 years - start date: 1/29/2016)

Here's the timeline for the election, we are giving ample time due to the holidays:

- Oct 26th: Nominations Open (TODAY)
- Nov 30th: Nominations Close
- Dec 1st: Qualification Period Opens
- Dec 14th: Qualification Period Closes (end of KubeCon Seattle)
- Dec 17th: Election Opens
- Jan 22nd: Election Closes
- Jan 29th: Election Results Posted!

In summary, these nominations are for the 6 slots nominated by CNCF members. You have to be a CNCF member in order to nominate someone. Each CNCF member CAN NOMINATION ONLY TWO PEOPLE MAXIMUM, where only one of them can come from your company (the other candidate can come from anywhere). See CNCF Charter Section 6(e) for full election process details: https://www.cncf.io/about/charter

**Comment by caniszczyk** [comment]
had a couple of CNCF members ask for TOC attendance records so posting it here:
https://docs.google.com/spreadsheets/d/1kRJLoWBOKQCPbCYV-YqdCqLG0-vBCt8qI3FNiIKoYtg/edit#gid=2000252942

**Comment by caniszczyk** [comment]
updated the process to reflect https://lists.cncf.io/g/cncf-toc/message/2580

**Comment by caniszczyk** [comment]
https://www.cncf.io/blog/2019/01/29/new-year-new-toc/
